funderfinder/sources/tidelift.py

```diff
@@ -39,7 +40,7 @@ def run(self, gh_project_slug: Union[str, None] = None) -> list:
                 "is_funded": False,
             }
         )
-        return [stats] if stats else []
+        return [stats] if stats["is_funded"] else []
```

Comment:
Otherwise the value is truthy and the wrapper script treats it as though a tidelift reference was found